packages/blob/src/index.ts

```diff
@@ -62,10 +62,11 @@ export interface HeadBlobResult {
   pathname: string;
   contentType: string;
   contentDisposition: string;
+  cacheControl: string;
 }
 
 interface HeadBlobApiResponse extends Omit<HeadBlobResult, 'uploadedAt'> {
-  uploadedAt: string;
+  uploadedAt: string; // when receiving data from our API, uploadedAt is a string
```

Comment:
shouldn't it be partial then?

Reply:
We do receive updatedAt but it's a string, and when we return it to the customer it becomes a Date so it's omit to change its type